src/service-api.ts

```diff
@@ -29,14 +29,23 @@ const plugin: FastifyPluginAsync = async (fastify) => {
     //   },
     // );
 
-    // get members
+    // get age categories
     fastify.get(
-      '/allcategories',
+      '/allcategories/age',
```

Comment:
I would rather have the following endpoint: `/categories/age`

Reply:
Actually is `age` the right word to talk about school levels?

Reply:
Good point, but actually I'm struggling with the naming... Don't have any good ideas in my mind now, but I think we can change this easily later on